main.py

```diff
@@ -1,10 +1,42 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Player.objects.all().delete()
+    Guild.objects.all().delete()
+
+    with open("players.json", "r") as players_file:
+        players_data = json.load(players_file)
+
+    list_of_players = list(players_data)
+
+    for player in list_of_players:
```

Comment:
It would be better to create `race` and `guid` separately, store them in variables, and then use them for `skill` and `player` creation.